app/views/solr_task/run_task.html.erb

```diff
@@ -0,0 +1 @@
+<h1> Your files have been indexed </h1>
```

Comment:
Would you make this a flash message, rather than its own page?

Reply:
if its a flash message @maxkadel on what page would you display it? the one that has the button?

Reply:
That's what I was thinking (display it on the page with the button), but I also think this is fine for a first pass.